package.json

```diff
@@ -71,11 +71,12 @@
     "raw-loader": "^3.1.0",
     "style-loader": "^1.0.0",
     "systemjs-webpack-interop": "^1.1.0",
-    "webpack": "^4.35.3",
-    "webpack-cli": "^3.3.5",
-    "webpack-dev-server": "^3.8.0"
+    "webpack": "^4.43.0",
+    "webpack-cli": "^3.3.11",
+    "webpack-dev-server": "^3.11.0"
   },
   "dependencies": {
+    "babel-jest": "^26.0.1",
```

Comment:
This should be in the `devDependencies`.

Reply:
@FlorianRappl Yes correct. Thanks for pointing out. Fixed it.